.eslintrc.js

```diff
@@ -6,20 +6,23 @@ module.exports = {
   },
   extends: [
     'airbnb-base',
+    'plugin:eslint-comments/recommended',
     'plugin:@typescript-eslint/recommended',
   ],
-  overrides: [],
   parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaVersion: 'latest',
     sourceType: 'module',
   },
   plugins: [
+    'eslint-comments',
     '@typescript-eslint',
     'unicorn',
   ],
   rules: {
     'comma-dangle': 'off',
+    'eslint-comments/no-unused-disable': 'error',
```

Comment:
Let me know what you think of this rule? Should it be enabled?

Reply:
I think it's a great rule – but seems to be included in `plugin:eslint-comments/recommended` already: https://github.com/mysticatea/eslint-plugin-eslint-comments/blob/master/lib/configs/recommended.js

Reply:
Oh yes didn't see that